shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-mysql/src/main/java/org/apache/shardingsphere/sql/parser/mysql/visitor/format/impl/MySQLFormatSQLVisitor.java

```diff
@@ -19,11 +19,14 @@
 
 import lombok.AccessLevel;
 import lombok.Getter;
+import lombok.Setter;
```

Comment:
This new function needs another PR, IMO.

Reply:
Yes. @tristaZero